src/jobs/retrigger.js

```diff
@@ -86,7 +86,7 @@ export default class RetriggerJob extends Base {
 
     console.log(`Handling retrigger for job ${taskId} in project '${project}'`);
     // Ensure when retrigger is sent that we use the right scopes for the job.
-    let scopes = projectConfig.scopes(this.config.try, project, true);
+    let scopes = projectConfig.scopes(this.config.try, project, false);
```

Comment:
nit: because of the default value on scopes() you could drop the 'false' argument altogether, but it is nice to be explicit.